### ISSUE TITLE
Validating a JS object against an InputType

### ISSUE THREAD

**Issue by abhiaiyer91** [graphql/graphql-js#974]
Hey there! I'm currently writing a form integration to Apollo-Client and was wondering if I could use an input type to validate a plain js object? This would be a cool way to handle this rather than use another library like joi or something!

**Comment by IvanGoncharov** [comment]
@abhiaiyer91 Yes, you can use [coerceValue](https://github.com/graphql/graphql-js/blob/master/src/execution/values.js#L193) for that. 
It's not officially exported by you can import if like:
```ts
import {coerceValue} from 'graphql/execution/values'
```
